package.json

```diff
@@ -24,6 +24,7 @@
     "@babel/eslint-parser": "workspace:*",
     "@babel/eslint-plugin-development": "workspace:*",
     "@babel/eslint-plugin-development-internal": "workspace:*",
+    "@babel/plugin-proposal-dynamic-import": "workspace:packages/babel-plugin-proposal-dynamic-import",
```

Comment:
Does that mean `@babel/node` should always be built after `@babel/plugin-proposal-dynamic-import`?

Reply:
No, it means I should have added the dependency from npm :sweat_smile: